es-spatial-plugin/project.clj

```diff
@@ -1,3 +1,4 @@
+#_{:clj-kondo/ignore [:unresolved-namespace]}
```

Comment:
what is the error here? which namespace can it not find?

Reply:
this is a project file, lets leave this here so that we know what kondo is all about.